Assets/AppCenter/Plugins/AppCenterSDK/Core/Android/AppCenterTaskWithResult.cs

```diff
@@ -21,7 +23,22 @@ public TResult Result
             {
                 // Locking in here is both unnecessary and can deadlock.
                 _completionEvent.WaitOne();
-                return _result;
+                if (_exception == null)
+                {
+                    return _result;
+                }
+                else
+                {
+                    throw _exception;
+                }
+            }
+        }
+        
+        public Exception Exception
```

Comment:
You can simplify this as below.
```
public Exception Exception => _exception;
```

Reply:
Looks like it has a compile error in unity saying 
`Feature `expression bodied members' cannot be used because it is not part of the C# 4.0 language specification`

Reply:
Oops.